frontend/src/metabase/core/components/Alert/Alert.styled.tsx

```diff
@@ -14,17 +14,17 @@ interface AlertRootProps {
 const colorsByVariant = {
   border: {
     info: "var(--mb-color-bg-medium)",
-    error: color("error"),
+    error: "var(--mb-color-error)",
```

Comment:
Update the error component colors according to [the design.](https://www.figma.com/design/W1TfzdrR8ldzp4vdjP1z36/SDK-Embedding%3A-admin-settings-%2B-embed-flow?node-id=11613-23031&t=z4vjXKUradcAyyAp-0)